src/ssl.c

```diff
@@ -20932,35 +22158,40 @@ int wolfSSL_CIPHER_get_bits(const WOLFSSL_CIPHER *c, int *alg_bits)
 
 int wolfSSL_sk_X509_NAME_num(const STACK_OF(WOLFSSL_X509_NAME) *s)
 {
-    (void) s;
     WOLFSSL_ENTER("wolfSSL_sk_X509_NAME_num");
-    WOLFSSL_STUB("wolfSSL_sk_X509_NAME_num");
 
-    return SSL_FAILURE;
+    if (s == NULL)
+        return -1;
+    return s->num;
```

Comment:
Yes gcc is happy with it, clang compilers treat it as a warning which gets treated as an error during make.